.github/workflows/build.yml

```diff
@@ -6,125 +6,64 @@ jobs:
-            gnuradio-dev \
-            libairspy-dev \
-            libairspyhf-dev \
-            libfreesrp-dev \
-            libhackrf-dev \
-            libusb-1.0-0-dev \
-            libsoapysdr-dev \
-            soapysdr-module-remote \
-            libuhd-dev \
-            liborc-0.4-dev \
-            libhidapi-dev
-
-          cd /tmp
-          git clone https://gitea.osmocom.org/sdr/rtl-sdr.git
-          cd rtl-sdr
-          git checkout v2.0.1
-          mkdir build
-          cd build
-          cmake -DCMAKE_INSTALL_PREFIX=/usr -DDETACH_KERNEL_DRIVER=ON ..
-          make -j4
-          make install
-          ldconfig
-
-          cd /tmp
-          git clone https://github.com/Nuand/bladeRF.git
-          cd bladeRF
-          mkdir build
-          cd build
-          cmake -DCMAKE_INSTALL_PREFIX=/usr ..
-          make -j4
-          make install
-          ldconfig
-
-          cd /tmp
-          git clone https://github.com/dl1ksv/gr-fcdproplus.git
-          cd gr-fcdproplus
-          mkdir build
-          cd build
-          cmake -DCMAKE_INSTALL_PREFIX=/usr ..
-          make -j4
-          make install
-          ldconfig
-
-          cd /tmp
-          git clone https://gitea.osmocom.org/sdr/libosmo-dsp.git
-          cd libosmo-dsp
-          autoreconf -i
-          ./configure --prefix=/usr
-          make -j4
-          make install
-          ldconfig
+          sudo apt-get update -qq
+          sudo apt-get install -y --no-install-recommends \
+            libopengl0 \
+            libegl1
 
-          cd /tmp
-          git clone https://gitea.osmocom.org/sdr/gr-iqbal.git
-          cd gr-iqbal
-          git checkout gr3.8
-          mkdir build
-          cd build
-          cmake -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_MODULES_DIR=/usr/lib/x86_64-linux-gnu/cmake ..
-          make -j4
-          make install
-          ldconfig
-
-          cd /tmp
-          git clone https://gitea.osmocom.org/sdr/gr-osmosdr.git
-          cd gr-osmosdr
-          git checkout origin/gr3.8
-          git cherry-pick -n 9c09c90d920dd4906fa8bb9d8443eef84d2565a3
-          mkdir build
-          cd build
-          cmake -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_MODULES_DIR=/usr/lib/x86_64-linux-gnu/cmake -DENABLE_PYTHON=False ..
-          make -j4
-          make install
-          ldconfig
+      - name: Install conda dependencies
+        id: setup-micromamba
+        uses: mamba-org/setup-micromamba@v1
+        with:
+          environment-name: gqrx
+          create-args: >-
+            c-compiler
+            cxx-compiler
+            cmake
+            make
+            pkg-config
+            gnuradio-core
+            gnuradio-osmosdr
+            libboost-headers
+            mesa-libgl-devel-cos6-x86_64
+            pybind11
+            pulseaudio
+            qt6-gtk-platformtheme
+            qt6-main
+            soapysdr-module-fcdpp
+            soapysdr-module-lms7
+            soapysdr-module-netsdr
```

Comment:
This may also be redundant, since the gnuradio-osmosdr package includes RFSpace / NetSDR support.